src/pages/home/report/ReportActionItem.tsx

```diff
@@ -646,16 +646,16 @@ function ReportActionItem({
             } else {
                 children = <ReportActionItemBasicMessage message={ReportUtils.getIOUSubmittedMessage(action)} />;
             }
-        } else if (ReportActionsUtils.isActionOfType(action, CONST.REPORT.ACTIONS.TYPE.APPROVED)) {
-            const wasAutoApproved = ReportActionsUtils.getOriginalMessage(action)?.automaticAction ?? false;
+        } else if (ReportActionsUtils.isUnapprovedOrApprovedAction(action)) {
```

Comment:
Let's split two conditions separately because the logic in two cases is different

Reply:
same to other places